controllers/tasks.js

```diff
@@ -0,0 +1,97 @@
+const Task = require('../models/Task')
+
+const addTask = (req, res) => {
+    res.render('pages/addTask')
+}
+
+const createTask = async (req, res) => {
+    try {
+        if (req.body.complete) {
+            req.body.completed = true
+        }
+        await Task.create(req.body);
+        req.session.pendingMessage = "The task was created."
+        res.redirect("/tasks")
+    } catch (err) {
+        if (err.name === "ValidationError") {
+            res.locals.message = Object.values(err.errors)
+                .map((item) => item.message)
+                .join(", ")
+        } else {
+            res.locals.message = "Something went wrong."
```

Comment:
Definitely `console.error(e)` on the catch-all path, or else you're setting up a recipe for confusion (especially in a larger codebase!)